charts/keycloak/values.yaml

```
@@ -155,17 +155,25 @@ livenessProbe: |
   httpGet:
     path: /auth/
     port: http
-  initialDelaySeconds: 300
+  initialDelaySeconds: 5
   timeoutSeconds: 5
 
 # Readiness probe configuration
 readinessProbe: |
   httpGet:
     path: /auth/realms/master
     port: http
-  initialDelaySeconds: 30
+  initialDelaySeconds: 5
   timeoutSeconds: 1
 
+# Startup probe configuration
+startupProbe: |
+  httpGet:
+    path: /auth/
+    port: http
+  initialDelaySeconds: 150
+  timeoutSeconds: 5
```

This will slow down startup a lot because it waits 150 s until the probe even kicks in, no matter how fast startup really is. Instead, we should increase the failure threshold.

```suggestion
  initialDelaySeconds: 30
  timeoutSeconds: 1
  failureThreshold: 30
  periodSeconds: 5
```